content/about.md

```diff
@@ -26,6 +26,20 @@ Additional information includes:
 - Related published papers
 - Related links (homepages, download sources, documentation, etc.)
 
-### Credits
+### Citing this Work
+
+If you would like to cite this overview in your (academic) work, we recommend to cite the exact release that the cited information refers to, e.g.,
+
+```
+@misc{idd100,
+  author = {{Intrusion Detection Datasets} contributors},
+  title = {{Intrusion Detection Datasets v1.0.0 -- GitHub}},
+  year = {2024},
+  howpublished = {\url{https://github.com/fkie-cad/intrusion-detection-datasets/releases/tag/v1.0.0}},
+  note = {[Online; accessed DD-MMM-YYYY]},
+}
+```
```

Comment:
I personally prefer `@online`, but don't know if its accessible everywhere
```
@online{idd100,
    author = {{Intrusion Detection Datasets} contributors},
    title = {{Intrusion Detection Datasets v1.0.0 -- GitHub}},
    year = {2024},
    url = {https://github.com/fkie-cad/intrusion-detection-datasets/releases/tag/v1.0.0},
    urldate = {YYYY-MM-DD},
}
```

Reply:
> I personally prefer `@online`, but don't know if its accessible everywhere

That's exactly the problem. BibLaTeX supports the `online` type by default. However, for example, the USENIX class omits both `url` and `urldate`, so I think using `misc` is the most compatible way.